lesson18/src/css/reset.css

```diff
@@ -0,0 +1,75 @@
+/*reset*/
```

Comment:
write reference for rerset.css

Reply:
Thank you for your review!
I commented that it is reset.css original.

[c509e00](https://github.com/nyattsu72/hanson-frontend-cource/pull/26/commits/c509e00de4955301996dbd9a0351f8d8d29b03f6)